### PR TITLE
Bug 1122328 - Add Finnish (fi), Armenian (hy-AM), Burmese (my) to master...

### PR DESCRIPTION
**Pull request by TheoChevalier**
... / Add Ewe (ee), Lingala (ln), Wolof (wo), Igbo (ig), Yoruba (yo), Hausa (ha) and locales to 2.0, add Wolof keyboard + generic keyboard for Africa to 2.0

### CODE REVIEW

**Comment by try-server-hook** [comment]
[TheoChevalier Théo Chevalier (TheoChevalier)](https://github.com/TheoChevalier) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=aa5dff5b91c6)

**Comment by try-server-hook** [comment]
[TheoChevalier Théo Chevalier (TheoChevalier)](https://github.com/TheoChevalier) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e55525506e66)

**Comment by try-server-hook** [comment]
[TheoChevalier Théo Chevalier (TheoChevalier)](https://github.com/TheoChevalier) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0d2a25f2e9c5)

File: apps/keyboard/js/layouts/en-Africa.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+Keyboards['en-Africa'] = {
+  label: 'English (Africa)',
+  shortLabel: 'En',
+  imEngine: 'latin',
+  types: ['text', 'url', 'email', 'password'],
```
Comment (RudyLu) [inline apps/keyboard/js/layouts/en-Africa.js]: for v2.0, we don't need password type here.

File: apps/keyboard/js/layouts/wo.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Keyboards.wo = {
+  label: 'Wolof',
+  shortLabel: 'Wo',
+  menuLabel: 'Wolof',
+  imEngine: 'latin',
+  types: ['text', 'url', 'email', 'password'],
```
Comment (RudyLu) [inline apps/keyboard/js/layouts/wo.js]: same here, for v2.0, remove password type here.

File: apps/keyboard/js/layouts/en-Africa.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        { value: '9' }, { value: '0' }
+      ], [
+        { value: '@' }, { value: '#' },
+        { value: '$', className: 'alternate-indicator' }, { value: '&' },
+        { value: '*' }, { value: '-' }, { value: '_' }, { value: '/' },
+        { value: '(' }, { value: ')' }
+      ], [
+        { value: 'Alt', ratio: 1.5,
+          keyCode: KeyboardEvent.DOM_VK_ALT,
+          className: 'page-switch-key',
```
Comment (RudyLu) [inline apps/keyboard/js/layouts/en-Africa.js]: remove className and targetPage here, these properties are not required in v2.0.

**Comment by try-server-hook** [comment]
[TheoChevalier Théo Chevalier (TheoChevalier)](https://github.com/TheoChevalier) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=701b679b4ed7)

**Pull request merged**